### DISCUSSION TITLE
How do i use the "ENABLE_PUBLIC_LIST" feature?

### DISCUSSION THREAD

**Discussion by petersem** [enchant97/hasty-paste#23] [Q&A] [answered]
Will this give me a list page to see all pastes?

**Comment by enchant97** [comment, marked as answer]
It's currently only accessible through the REST API by making a GET request to: `example.com/api/pastes/`. I will be implementing a public list page in the future, also with the ability to mark a paste as public or "private".

**Reply by enchant97** [reply]
As an aside, you can also now (v1.4.0) list the pastes using the new experimental CLI. Instructions on how to access are in the documentation under "Usage".
